scripts/publish.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+#
+# Little script to publish to crates.io
+# Usage: publish.sh x.y.z
+#
+# Dependencies: git, ripgrep
```

Comment:
I'm a bit lost where `cargo set-version` is used in this script :sweat_smile:

Reply:
Haha oops, sorry it's only used in the other one! :smile: